python/arcticdb/vector_db/vector_db.py

```diff
@@ -434,6 +436,42 @@ def top_k(
         result.index = list(result.index[:-1]) + ["similarity"]
         return result
 
+    def top_k(
+        self,
+        namespace: str,
+        query_vector: Collection[float],
+        k: int,
+        norm: Optional[str] = None,
+        index: Optional[str] = None,
+        include_vectors: bool = False,
+        include_metadata: bool = False,
+    ) -> List[Dict[str, Union[str, pd.Series]]]:
+        """
+        Returns the top `k` most similar vectors to `query_vector` in `namespace`. We use the vector ID as a tiebreaker.
+
+        Parameters
+        ----------
+        namespace
+            The namespace in which to perform the top-k search.
+        k
+            The number of vectors to return from the search.
+        query_vector
+            The vector from which distances are taken.
+        norm
+            The distance norm based on which the top k will be computed. This is presently ignored; the default
+            is Euclidean.
+        index
+            The index based on which to compute the top k. This is also ignored.
+        include_vectors
```

Comment:
Does pinecone have an `include_distance` parameter as well? Or is that always included?

Reply:
No (https://docs.pinecone.io/reference/query)